faker/providers/phone_number/pt_PT/__init__.py

```diff
@@ -9,6 +9,7 @@ class Provider(PhoneNumberProvider):
         '(351) 93# ### ###',
         '(351) 96# ### ###',
         '(351) 2## ### ###',
+        '(351) 2# ### ### #',
```

Comment:
while we're here, shouldn't all those be `+351` instead of `(351)`?

Reply:
+351 is equal to (351) basicly, we can use only the +351 however im only following the already created content

Reply:
Looking at other providers, I think we should have both, unless it's really uncommon to see `+351` in Portugal.

Do you feel like adding `+351` versions of all the `(351)` patterns? That is, unle